src/query/execute_schema.c

```diff
@@ -8532,7 +8532,7 @@ do_create_entity (PARSER_CONTEXT * parser, PT_NODE * node)
   PT_NODE *create_index = NULL;
   DB_QUERY_TYPE *query_columns = NULL;
   PT_NODE *tbl_opt = NULL;
-  bool found_reuse_oid_option = false, reuse_oid = true;
+  bool reuse_oid = true;
```

Comment:
the default value is true but at the below case clause, what if the tbl_opt->info.table_option.option is not PT_TABLE_OPTION_REUSE_OID and not PT_TABLE_OPTION_DONT_REUSE_OID ? the value reuse_oid = true is right ?

Reply:
Since already adding PT_NODE for table option related reuse_oid into table_option_list at semantic check routine if creating a table without table option, there is not this case.
So, I think that that reuse_oid is true or false is no problem.